### ISSUE TITLE
OutlineEffect with xray is never in front of objects as of 6.24.0

### ISSUE THREAD

**Issue by canadaduane** [vanruesc/postprocessing#351]
### Description of the bug

As of 6.24.0, when a small object that is part of the `OutlineEffect.selection` is in front of any large object (not a part of the selection), OutlineEffect always treats the small object's edges as if they are 'hidden' and uses `hiddenEdgeColor`. This appears to be a regression since 6.23.5.

### To Reproduce

I'm using the following outline effect options:

```
    this.outlineEffect = new OutlineEffect(this.scene, this.camera, {
      blendFunction: BlendFunction.ALPHA,
      edgeStrength: 5,
      visibleEdgeColor: 0xffffff,
      hiddenEdgeColor: 0x5f5f5f,
      blur: true,
      xRay: true,
    });
```

### Expected behavior

The behavior in 6.23.5 seems correct to me, where the outline is not treated as "hidden" when not occluded by objects.

### Screenshots

6.23.5:

![image](https://user-images.githubusercontent.com/129/155898545-3766628c-8f7a-4d29-985e-662a50ca9d94.png)

6.24.1:
![image](https://user-images.githubusercontent.com/129/155898629-2dd1ddbd-b62a-42b0-842a-05db83bb558d.png)

### Library versions used

 - Three: 0.131.3
 - Post Processing: 6.23.5 / 6.24.1

### Desktop

 - OS: Linux
 - Browser: Firefox
 - Graphics hardware: Intel

**Comment by vanruesc** [comment]
Thanks for the report, can confirm that it's broken. I'll investigate when I get a chance.

**Comment by vanruesc** [comment]
Fixed in `postprocessing@6.25.0`.
